### PR TITLE
Create PatientCareDB and its ER Diagram

### PR DESCRIPTION
**Pull request by jicamargo**
# Create PatientCareDB and its ER Diagram
This PR includes the initial setup for the PatientCareDB PostgreSQL database and the creation of a database diagram to visualize the database structure.

## Changes Made

- Created the PatientCareDB PostgreSQL database.
- Defined tables for 'patients', 'medical_histories', 'invoices', 'treatments', 'invoice_items', and 'medical_history_treatments'.
- Added columns and data types for each table.
- Established primary keys and foreign key relationships between tables.
- Designed and created a database diagram to visualize the database structure.

## Database Diagram

image

## Checklist

- [ ] Database setup completed.
- [ ] Database schema design implemented.
- [ ] Database diagram created and included in the PR.
- [ ] Testing and validation of the database functionality.
- [ ] Documentation updated with the new database details.
- [ ] Code reviewed and approved.

## Peer programmers
This project was build by:
  - @benja27 
  - @sylvester-wamaya 
  - @jicamargo 

Thank you for your review and suggestions to improve this project 👍🏼

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    quantity INT,
+    total_price DECIMAL,
+    invoice_id INT,
+    treatment_id INT,
+    FOREIGN KEY (invoice_id) REFERENCES invoices(id),
+    FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
+
+-- Create the 'medical_history_treatments' table (Many-to-Many relationship)
+CREATE TABLE medical_history_treatments (
```
Comment (BregornOriginal) [inline schema_based_on_diagram.sql:50]: Good work implementing a Many-to-Many relationship! 💯

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    treatment_id INT,
+    medical_history_id INT,
+    FOREIGN KEY (treatment_id) REFERENCES treatments(id),
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+    PRIMARY KEY (treatment_id, medical_history_id)
+);
```
Comment (BregornOriginal) [inline schema_based_on_diagram.sql:56]: - [ ] Kindly as was requested, `CREATE INDEX` for each necessary `id`. For example: `CREATE INDEX ON medical_histories (patient_id);`

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Team

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

hand-sketching-stair-slogan-one-step-closer-chalkboard-79948193

## Highlights

✅ Linters are passing
✅ Good commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jicamargo** [comment]
Thank you @BregornOriginal, for your suggestions.

We create add 6 indexes to improve the performance:
CREATE INDEX idx_medical_histories_patient_id ON medical_histories (patient_id);
CREATE INDEX idx_invoices_medical_history_id ON invoices (medical_history_id);
CREATE INDEX idx_invoice_items_invoice_id ON invoice_items (invoice_id);
CREATE INDEX idx_invoice_items_treatment_id ON invoice_items (treatment_id);
CREATE INDEX idx_medical_history_treatments_treatment_id ON medical_history_treatments (treatment_id);
CREATE INDEX idx_medical_history_treatments_medical_history_id ON medical_history_treatments 

Changes were added to schema_based_on_diagram.sql

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi team , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done working on the requested changes.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**